app/components/post_attachment_opengraph/post_attachment_opengraph.js

```diff
@@ -96,10 +95,22 @@ export default class PostAttachmentOpenGraph extends PureComponent {
         });
 
         if (imageUrl) {
-            ImageCacheManager.cache(null, imageUrl, this.getImageSize);
+            ImageCacheManager.cache(this.getFilename(imageUrl), imageUrl, this.getImageSize);
         }
     }
 
+    getFilename = (link) => {
```

Comment:
Non-blocking but maybe good to have test particularly on this function.

Reply:
Ticket created to deal with this separately https://mattermost.atlassian.net/browse/MM-12016

Reply:
Great idea. If we see code that isn't tested, but should be, we should just default to creating a quick ticket for it. I wonder if we could set up a template to make that really easy